content/react/concepts/routing/routing.md

```diff
@@ -21,27 +21,41 @@ React Router is a third-party library that allow components to be used as routed
 - `react-router-native`: React Native version
 - `react-router-dom`: Recommended for web applications
 
-Install with `npm`:
+Install the newest version with `npm`:
 
 ```
-npm install react-router-dom
+npm install react-router-dom@latest
 ```
 
+The newest version of React Router is v6, which comes with many breaking changes including:
+
+- Using a `<Routes />` component instead of a `<Switch />` component for `<Route />` configuration.
+- Using the `element` attribute instead of `component` when passing a JSX to a `<Link />`.
+
 ## Example
 
-Below, a `BrowserRouter` is used to map the navigation `<Link/>` components to a `<Switch/>` component. The `<Switch/>` renders the appropriate `<Route>` based on a top-down search and match between the `to` attribute in the `Link` and the `path` attribute in the `Route`.
+Below, a `<BrowserRouter/>` (aliased as `<Router/>`) is used to map the navigation `<Link/>` components to a `<Routes/>` component. The `<Routes/>` renders the appropriate `<Route>` based on the most specific-matching `path` among all possible matches .
```

Comment:
We can, however, include the distinct changes like how the order that the route paths are listed no longer takes precedence over the most specific-matching path.

```suggestion
Below, a `<BrowserRouter/>` (aliased as `<Router/>`) is used to map the navigation `<Link/>` components to a `<Routes/>` component. The `<Routes/>` renders the appropriate `<Route>` based on the most specific-matching `path` among all possible matches.
```

Extra space before the period. If you want to include my point mentioned above, it can be added at the end of the sentence or as a **Note**: In older versions, the order that the route paths are listed took precedence over the most specific-matching path.

This note can still be helpful for people transitioning to newer version